### PR TITLE
CE-3199: Move global shortcuts entry point to wikia bar

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Move global shortcuts entry point to wikia bar

### CODE REVIEW

File: includes/wikia/services/usercommands/ShortcutsUserCommand.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/*
+ * Displays Global Shortcuts in toolbar
+ */
+class ShortcutsUserCommand extends UserCommand {
+    
+    protected $overflow = false;
+    protected $defaultRenderType = 'link';
+    
+    protected function buildData() {
+        $this->available = F::app()->wg->user->isLoggedIn();
```
Comment (adamkarminski) [inline includes/wikia/services/usercommands/ShortcutsUserCommand.php]: I think it's unnecessary to call it twice, better cache the result in a local variable.

File: includes/wikia/services/UserCommandsService.class.php [hunk 2]
```diff
@@ -34,7 +34,7 @@ public function get( $name, $options = array() ) {
 						}
 						break;
 					case "Action":
-						if (in_array($data,array("CustomizeToolbar", "DevInfo"))) {
+						if (in_array($data,array("CustomizeToolbar", "DevInfo", "Shortcuts"))) {
```
Comment (adamkarminski) [inline includes/wikia/services/UserCommandsService.class.php]: Single quotes `'...'`

EDIT: And spaces. :)

File: includes/wikia/services/usercommands/ShortcutsUserCommand.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        $this->enabled = F::app()->wg->user->isLoggedIn();
+        $this->caption = 'Shortcuts';
```
Comment (adamkarminski) [inline includes/wikia/services/usercommands/ShortcutsUserCommand.php]: Shouldn't it be localized?

Comment (kamilkoterba) [inline includes/wikia/services/usercommands/ShortcutsUserCommand.php]: +1

**Comment by kamilkoterba** [comment]
Action Shortcuts should be dependent on extension enabling var. 
Let's use $wgEnableGlobalShortcutsExt

**Comment by kamilkoterba** [comment]
LGTM

**Comment by Grunny** [comment]
LGTM.

**Comment by azarek** [comment]
LGTM :)

**Pull request merged**